### PR TITLE
[WIP] Renaming of cartesian link speed fields related to #113

### PR DESCRIPTION
**Pull request by Thieso**
PR #113 was merged but had wrong field names so this PR is supposed to correct for these incorrect names.

### CODE REVIEW

File: msg/MotionPlanRequest.msg [hunk 1]
```diff
@@ -51,8 +51,8 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
-# Maximum cartesian speed for the given end effector.
+# Maximum cartesian speed for the given link.
 # If max_cartesian_speed <= 0 the trajectory is not modified.
 # These fields require the following planning request adapter: default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed
-string cartesian_speed_end_effector_link
+string cartesian_speed_link
```
Comment (felixvd) [inline msg/MotionPlanRequest.msg:57]:
```suggestion
string cartesian_speed_limited_link
```
While we're renaming, why not make it a little easier to understand?

File: srv/GetCartesianPath.srv [hunk 2]
```diff
@@ -43,6 +43,13 @@ bool avoid_collisions
 # Specify additional constraints to be met by the Cartesian path
 Constraints path_constraints
 
+# Maximum cartesian speed for the given link.
+# If max_cartesian_speed <= 0 the trajectory is not modified.
+# WARNING NOT FUNCTIONAL YET.
```
Comment (felixvd) [inline srv/GetCartesianPath.srv:48]: You can remove these comments from the commits and mark your PR as dependent or WIP instead.

**Review by felixvd — APPROVED** [review]
Just some suggestions to go with the other review. Great stuff.

File: msg/MotionPlanRequest.msg (hunk 1 above)
Comment (Thieso) [reply msg/MotionPlanRequest.msg]: Good point, thanks

File: msg/MotionPlanRequest.msg [hunk 3]
```diff
@@ -51,8 +51,8 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
-# Maximum cartesian speed for the given end effector.
+# Maximum cartesian speed for the given link.
 # If max_cartesian_speed <= 0 the trajectory is not modified.
 # These fields require the following planning request adapter: default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed
```
Comment (gautz) [inline msg/MotionPlanRequest.msg:56]: ``default_planner_request_adapters/setMaxCartesianLinkSpeed``
btw I think it should be changed to **S**etMaxCartesianLinkSpeed (capital S) in https://github.com/ros-planning/moveit/pull/2674 for consistency with the names of other planner_request_adapters

**Comment by gautz** [comment]
@felixvd @v4hn what about merging this before the summer break? The corresponding moveit PR could be merged directly too.

File: msg/MotionPlanRequest.msg (hunk 3 above)
Comment (gautz) [reply msg/MotionPlanRequest.msg:56]: @Thieso We forgot one last detail in the comments:
default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed -> default_planner_request_adapters/LimitMaxCartesianLinkSpeed

Comment (Thieso) [reply msg/MotionPlanRequest.msg]: Good catch, thanks.

**Comment by v4hn** [comment]
Replaced by #130

**Pull request closed without merging**